src/ProjectTemplates/Shared/TemplatePackageInstaller.cs

```diff
@@ -128,8 +128,8 @@ private static async Task InstallTemplatePackages(ITestOutputHelper output)
 
         private static async Task VerifyCanFindTemplate(ITestOutputHelper output, string templateName)
         {
-            var proc = await RunDotNetNew(output, $"");
-            if (!proc.Output.Contains($" {templateName} "))
+            var proc = await RunDotNetNew(output, $"--list");
+            if (!(proc.Output.Contains($" {templateName} ") || proc.Output.Contains($",{templateName}") || proc.Output.Contains($"{templateName},")))
```

Comment:
Could you quickly summarize why these lines in the installer need to change❔ For example has a bare `dotnet new` command been deprecated at the same time the output format changed❔

Reply:
> For example has a bare `dotnet new` command been deprecated at the same time the output format changed❔

No clue, I'm testing to see if that's the case by using `--list`.

The first part with checking commas is because `dotnet new` now displays all "short names" for the template. Might be able to clean it up a bit, but again, doing exploratory testing here.

Reply:
Huh. I don't remember a breaking change notification but the output might not be something we should really rely on. @KathleenDollard who should we loop in here❔

Reply:
Looks like `--list` worked, now just failing on some restore issues.

Reply:
Old `dotnet new` output would list all templates it looks like, new `dotnet new` lists a predetermined list of "common templates".

Looks like a very deliberate change, so I think our reaction to add `--list` is fine.

Reply:
I think the `or`s I added can be removed now that `--list` is used

Reply:
I don't think so, I checked and it was displaying templates the way your are testing them. For me the --list would just display a new prompt, and then the same table as before without the --list.